.gitignore

```diff
@@ -81,21 +81,26 @@ src/gregorio
 #emacs auto gen files
 auto/
 
-!UserManual.pdf
+#Debian files
+*.changes
+*.xz
+*.dsc
+*.deb
 
 #gregorio specific
-
+doc/UserManual.pdf
```

Comment:
I don't think this line is needed.  Doesn't line 48 already exclude all `.pdf` files?